readme.md

```diff
@@ -64,7 +92,11 @@ Your Answer:
 Describe the purpose of views and models in FE JS.
 
 Your Answer:
-```text
+```
+Models hold the constructor function and fetch from the database.
+
+Views hold the prototype function and create a user-friendly not crypted view of the json
```

Comment:
This is generally correct. Only things I'd clarify are: models fetch / update data in the database, but not directly. Instead, they use AJAX requests to request data from the backend application (which in turn gets it from the database).

Front end views do turn JSON into HTML, but also update the dom and respond to user events (clicks, key presses, etc)